b-lab-org/ember-cli-impact-core
# [Bug Fix] Circle component not removing tooltips when destroyed

## Overview
This was something I noticed when doing QA on another PR, the `impact-circle` component was not cleanup up after itself.

There was also some fallout from #128, where that fix prevented viewing the dummy app locally.

## How to test
- `ember s`
- navigate to the widgets page with dev tools open
- note all of the `material tooltip` divs placed into the DOM
- navigate away
- note that they're all cleaned up
